src/status_im/contexts/wallet/send/save_address/style.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.contexts.wallet.send.save-address.style)
+
+(def title-input-container
+  {:padding-horizontal 20
+   :padding-top        12
+   :padding-bottom     16})
+
+(def account-avatar-container
+  {:padding-horizontal 20
+   :padding-top        12})
+
+(def divider-1
+  {:margin-bottom 12})
+
+(def color-picker-container
+  {:padding-vertical 12})
+
+(def section-container
+  {:padding-horizontal 20
+   :padding-bottom     4})
+
+(def color-picker
+  {:padding-horizontal 20
+   :padding-top        12})
+
+(def divider-2
+  {:padding-top    8
+   :padding-bottom 12})
+
+(def save-address-button
+  {:padding-horizontal 20})
+
+(def data-item
+  {:margin-horizontal 20
+   :padding-vertical  8})
+
+(def address-container
+  {:padding-top    8
+   :padding-bottom 12})
+
+(def container
+  {:flex         1
+   ;; This negative margin is needed because bottom sheet as a modal
+   ;; doesn't play well with `:sheet? true`
+   :margin-top   -20
```

Comment:
I have a couple of questions about this 🤔 
* Is it possible to make `20` or `-20` a variable that's shared?
* Is there other code that also does this?

Reply:
@flexsurfer faced this issue recently too I believe. Not sure he handled it though

Reply:
it's not related to the sheet or modal, could you please remove the comment, I fixed that in my PR but it's not merged yet https://github.com/status-im/status-mobile/pull/19088/files#diff-6d74d88ad4426a5925a653f48ad07d726a210c6f4fdc8987fe22ce2faa58a8ceR56

Reply:
@flexsurfer's fix will work. I'll remove this and add the `top?` prop. It will break my PR, but everything will start working when Andrey's PR is merged.

Reply:
sorry i just removed that, because it breaks floating button, we need to think on the proper solution

Reply:
sa basically in my PR i have a similar screen, and I don't need any margins, could you show that's the problem?

Reply:
This is my margin problem.

![Simulator Screenshot - iPhone 13 - 2024-03-14 at 14 58 40](https://github.com/status-im/status-mobile/assets/1925158/fb6a0791-1974-4481-97a9-3fb5f943503e)

Reply:
you need to remove this line https://github.com/status-im/status-mobile/pull/19079/files#diff-b1cfd4ba26cb8e7c23e291f1687cfeff668ff227edee9e3feeeae99b6b3fdbb8R60